.github/workflows/ci.yml

```diff
@@ -0,0 +1,158 @@
+name: CI
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+      - develop
+      - "release/**"
+env:
+  APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
+  APPLITOOLS_BATCH_ID: ${{ github.event.pull_request.head.sha }}
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      deploy: ${{ steps.filter.outputs.deploy }}
+      vrt: ${{ steps.filter.outputs.vrt }}
+    steps:
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
```

Comment:
TODO: Figure out how we can dial these in a bit more

Reply:
These filters seem to miss the edge cases we discussed around babel changes or storybook configuration changes in upstream packages?